aspnetcore/security/authentication/index.md

```diff
@@ -0,0 +1,103 @@
+
+In some cases, the call to `AddAuthentication` is automatically made by other extension methods. For example, when using [ASP.NET Core Identity](xref:security/authentication/identity),`AddAuthentication` is called. If multiple schemes are used, authorization policies (or authorization attributes) can [specify the authentication scheme (or schemes)](xref:security/authorization/limitingidentitybyscheme) they depend on to authenticate the user. In the example above, the cookie authentication scheme could be used by specifying its name (`CookieAuthenticationDefaults.AuthenticationScheme` by default, though a different name could be provided when calling `AddCookie`).
+
+The Authentication middleware is added in `Startup.Configure` by calling the <xref:Microsoft.AspNetCore.Builder.AuthAppBuilderExtensions.UseAuthentication*> extension method on the app's `IApplicationBuilder`. This registers the  middleware which uses the previously registered authentication schemes. Call `UseAuthentication` before any middleware that depends on users being authenticated. When using endpoint routing, the call to `UseAuthentication` must go:
+
+* After `UseRouting`, so that route information is available for authentication decisions.
+* Before `UseEndpoints`, so that users are authenticated before accessing the endpoints.
+
+## Authentication Concepts
+
+### Authentication scheme
+
+An authentication scheme is a name which corresponds to:
+
+* An authentication handler. 
+* Options for configuring that specific instance of the handler. 
+ 
+Schemes are useful as a mechanism for easily referring to the authentication, challenge, and forbid behaviors of the associated handler. For example, an authorization policy can specify by name which authorization scheme (or schemes) should be used to authenticate the user. When configuring authentication, it's common to specify the default authentication scheme. The default scheme is used unless a resource requests a specific scheme. It's also possible to:
+
+* Specify different default schemes to use for authenticate, challenge, and forbid actions.
+* Combine multiple schemes into one using [policy schemes](xref:security/authentication/policyschemes).
+
+### Authentication handler
+
+An authentication handler:
+
+* Is a type that implements the behavior of a scheme.
+* Is derived from <xref:Microsoft.AspNetCore.Authentication.IAuthenticationHandler> or <xref:Microsoft.AspNetCore.Authentication.AuthenticationHandler>.
+* Has the primary responsibility to authenticate users.
+
+Based on the authentication scheme's configuration and the incoming request context, authentication handlers:
+
+* Construct <xref:Microsoft.AspNetCore.Authentication.AuthenticationTicket> objects representing the user's identity if authentication is successful.
```

Comment:
This is the only place I can think to put an HttpContext.AuthenticateAsync reference. Do we need a dedicated Authenticate section to mirror Challenge and Forbid?

Reply:
Yea, probably an 'Authenticate' section is a good idea (to mirror Challenge and Forbid). I'll add one.

Reply:
Ok, I've added an Authenticate section. @Tratcher, can you take a quick look?